docs/content/SplitPageLayout.mdx

```diff
@@ -0,0 +1,313 @@
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### With pane hidden on narrow viewports
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane visible={{narrow: false}}>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### With content hidden on narrow viewports
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane>
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content visible={{narrow: false}}>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### Without dividers
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header divider="none">
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Pane divider="none">
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Footer divider="none">
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### With pane on right
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
+  </SplitPageLayout.Header>
+  <SplitPageLayout.Content>
+    <Placeholder label="Content" height={240} />
+  </SplitPageLayout.Content>
+  <SplitPageLayout.Pane position="end">
+    <Placeholder label="Pane" height={120} />
+  </SplitPageLayout.Pane>
+  <SplitPageLayout.Footer>
+    <Placeholder label="Footer" height={64} />
+  </SplitPageLayout.Footer>
+</SplitPageLayout>
+```
+
+### Without padding
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header padding="none">
```

Comment:
nit: Do we need custom padding on every child or could we match the padding prop on the parent like [PageLayout](https://primer.style/react/PageLayout#with-condensed-spacing)?

`padding` prop on every child feels a lot like system props, this looks like you can customise the padding yourself with a value like `padding={0}` or `padding={1}`